Features/aperta 9726 card config checkbox

Bestra opened the pull request:
JIRA issue: https://jira.plos.org/jira/browse/APERTA-9726

#### What this PR does:
Allows users to add checkboxes to card config cards.  Setting 'default' values will be done as part of a separate Jira card that I'll link to here.
Saving checkbox answers will come for free as part of #2988 

Screenshots:
![screen shot 2017-04-06 at 3 29 33 pm](https://cloud.githubusercontent.com/assets/2043348/24772036/e132d6d4-1add-11e7-963a-39ff187ae5bd.png)
![screen shot 2017-04-06 at 3 29 26 pm](https://cloud.githubusercontent.com/assets/2043348/24772037/e1388642-1add-11e7-88a7-6eaa8e03327c.png)



#### Major UI changes

Added checkbox UI elements

---

#### Code Review Tasks:

Author tasks:
- [X] If I created a migration, I updated the base data.yml seeds file. [instructions](https://developer.plos.org/confluence/display/TAHI/Seeds+maintenance)

Reviewer tasks:

- [x] I skimmed the code; it makes sense
- [x] I read the code; it looks good
- [x] I ran the code (in the review environment or locally)
- [x] I performed a 5 minute walkthrough of the site looking for oddities
- [x] I have found the tests to be sufficient
- [x] I like the CHANGELOG entry
- [x] I agree the code fulfills the Acceptance Criteria
- [x] I agree the author has fulfilled their tasks

#### After the Code Review:

Author tasks:

- [ ] The Product Team has reviewed and approved this feature

Bestra committed:
Add label to card_contents

Bestra committed:
Update styles for card content check box

Bestra committed:
Add card config maintenance doc

Bestra committed:
Update data.yml for label column

diiq reviewed (approved):
Looks good. My only comment was on the maintenance guide. I'm not sure about where you've put it or how you've formatted it -- but do what thou wilt shall be the whole of the law.

Bestra committed:
Move card content doc to confluence

https://confluence.plos.org/confluence/display/TAHI/Card+Content+Type+Maintenance+Guide
